### PR TITLE
Add Subscription List page, Details page (partial), Modify banner,footer view, etc

### PR DESCRIPTION
**Pull request by sajibcse68**

### CODE REVIEW

File: js/apps/app.js [hunk 1]
```diff
@@ -35,14 +37,16 @@ global.clearAllHtmlContent = ()=> {
...
+  global.$lobbyModal.modal('hide');
 };
 if (window.location.search !== '') {
   const urlArr = window.location.search.split('=');
   window.location.replace('/lobby/#/token_id/' + urlArr[urlArr.length - 1]);
 }
 apiClient.auth.v1beta1.userService.whoAmI()
   .then(resp => {
-    Backbone.history.start({pushState: true, root: '/'});
+    Backbone.history.start({pushState: false, root: '/'});
```
Comment (a8uhnf) [inline js/apps/app.js]: don't we need `pushstate: true`?

File: js/views/lobby/invoice-list.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    apiClient.billing.v1beta1.invoiceService.list()
+      .then(apiLib.checkStatus)
+      .then(resp => {
+        const nowUrl = Backbone.history.getFragment();
+        if (nowUrl === that.url) {
+          headerAndFooter.addTopMenu();
+          headerAndFooter.addFooter();
+          const banner = new Banner();
+          banner.render();
+          console.log('resp: >>>', resp);
```
Comment (a8uhnf) [inline js/views/lobby/invoice-list.js]: need to remove console.log

File: js/views/lobby/subscription-details.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+              o.uid = subs.uid;
+              o.startAt = that.time.getTime(subs.start_at);
+              o.endAt = that.time.getTime(subs.end_at);
+              o.numUser = subs.spec.quotas['pkg.user'];
+              $(div).append(global.nunjucksEnv.render(that.subscriptionListRow, o));
+              fragment.appendChild(div);
+            }
+            $('#subscription-list').html(fragment);
+          }
+          */
```
Comment (a8uhnf) [inline js/views/lobby/subscription-details.js]: comment-out code

File: js/apps/app.js (hunk 1 above)
Comment (sajibcse68) [inline js/apps/app.js]: Yes, we need. But we can make it false to speed up developing. Finally, make `pushState: true`.

File: js/views/lobby/invoice-list.js (hunk 2 above)
Comment (sajibcse68) [inline js/views/lobby/invoice-list.js]: console.log here to get rid of error!

File: js/views/lobby/subscription-details.js (hunk 3 above)
Comment (sajibcse68) [inline js/views/lobby/subscription-details.js]: API call, WIP.

**Pull request merged**